src/PostgREST/App.hs

```diff
@@ -305,7 +315,6 @@ app dbStructure conf apiRequest =
       fldNames = fieldNames <$> readReq
       readDbRequest = DbRead <$> readReq
       mutateDbRequest = DbMutate <$> (mutateRequest apiRequest =<< fldNames)
-      rpcQParams = readRpcRequest apiRequest
```

Comment:
It's great that this is unified in the payload now. Probably should delete `iRpcQParams` from the `ApiRequest` data type as well.

Reply:
`iRpcQParams` deleted.